contentcuration/contentcuration/static/js/bundle_modules/settings.js

```diff
@@ -0,0 +1,8 @@
+// include all logic in "base" entrypoint
```

Comment:
Module makes it possible to get the dialog package on python/html-driven templates